bin/dropsonde

```diff
@@ -5,6 +5,9 @@ require 'dropsonde'
 class Dropsonde
   extend GLI::App
 
+  @cache = nil
+  @session = Dropsonde.new.puppet_db
```

Comment:
where is this `@session` coming from?

Reply:
In the previous implementation, `pdbclient` was a class variable which raised rubocop offenses. The `@session` variable stores the puppet_db connection which is used by all plugins to interact with `puppetdb`.

Reply:
Maybe the name could create some confusions...